scripts/apps/authoring/authoring/directives/SendItem.tsx

```diff
@@ -419,11 +424,12 @@ export function SendItem($q,
              * @param {Boolean} sendAllPackageItems - True to include all contained items for packages
              * @return {Object} promise
              */
-            function runSend(open, sendAllPackageItems) {
+            function runSend(open, sendAllPackageItems, _deskId, _stageId) {
                 scope.loading = true;
                 scope.item.sendTo = true;
-                var deskId = scope.selectedDesk._id;
-                var stageId = scope.selectedStage._id || scope.selectedDesk.incoming_stage;
+
+                const deskId = _deskId || scope.selectedDesk._id;
```

Comment:
there's only one place `runSend` is called and `deskId`/`stageId` are passed as an arguments. Is there a reason to have fallbacks?

I see that `stageId` already had a fallback to `scope.selectedDesk.incoming_stage`, but it's not used in scope.send. How about using that fallback there instead?